.github/workflows/test.yml

```diff
@@ -0,0 +1,15 @@
+name: test
+on: push
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+      - run: |
+          curl -fsSL get.nextflow.io | bash
+          sudo mv nextflow /usr/local/bin
```

Comment:
FYI, we will probably adopt this new action soon: https://github.com/MillironX/setup-nextflow

Notably, it allows `version: latest-everything` which installs either the latest stable release or the latest edge release, depending on which is newer (which is annoyingly difficult to do otherwise).

Not saying that we need this here, but just reminded me of it's existence and thought I'd mention it :)

Reply:
let's have some fun then 😉

Reply:
Yup, here we go: https://github.com/nf-core/setup-nextflow